scripts/management/query_mgmt_db.py

```diff
@@ -20,17 +20,17 @@ def print_run_status(db, run_name, error=False):
         for statum in status:
             print ''' Run_name: {}\n Process: {}\n Status: {}\n Job-ID: {}\n Last_Modified: {}\n Error: {} \n'''.format(*statum)
     else:
-        db.execute('''SELECT state.run_name, proc_type_enum.proc_type, status_enum.state, state.job_id, datetime(last_modified,'unixepoch')
+        db.execute('''SELECT state.run_name, proc_type_enum.proc_type, status_enum.state, state.job_id, datetime(last_modified,'unixepoch'), state.retries
                     FROM state, status_enum, proc_type_enum
                     WHERE state.proc_type = proc_type_enum.id AND state.status = status_enum.id
                             AND UPPER(state.run_name) LIKE UPPER(?)
                     ORDER BY state.run_name, status_enum.id
                     ''', (run_name,))
         status = db.fetchall()
-        print "{:>25} | {:>15} | {:>10} | {:>8} | {:>20}".format('run_name', 'process', 'status', 'job-id', 'last_modified')
+        print "{:>25} | {:>15} | {:>10} | {:>8} | {:>20} | {:>10}".format('run_name', 'process', 'status', 'job-id', 'last_modified', 'retries')
         print '_' * (25 + 15 + 10 + 20 + 8 + 3 * 4)
```

Comment:
Don't you need to make this longer when other lines are longer now? (I'm just being pedantic)